src/methods/using_methods/hard/q7/MainTest.java

```diff
@@ -0,0 +1,55 @@
+package methods.using_methods.hard.q7;
+
+import global.BaseTest;
+import global.tools.CustomAssertions;
+import global.tools.TestOption;
+import global.utils.MethodUtil;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons
+    public Clause[] testSentence(){
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "4.5";
+        return new Clause[]{
+                new StringLiteral("Please enter the length of a side:"),
+                new NewLine(),
+                new StringLiteral("The volume of the cube is "),
+                new DoubleLiteral("volume")
+        };
+    }
+
+    public void runMain(){VolumeOfCube.main(new String[0]);}
+
+    static Stream<Arguments> cubeVolumeInputProvider(){
+        return Stream.of(Arguments.of(0, 0.0), Arguments.of(1, 1.0), Arguments.of(-1.0, -1.0), Arguments.of(-6.0, -1.0), Arguments.of(-23.8, -1.0), Arguments.of(25.5, 16581.375), Arguments.of(5945.8, 2.1019911829991202E11), Arguments.of(0.34, 0.03930400000000001));
```

Comment:
```suggestion
        return Stream.of(Arguments.of(0, 0.0), Arguments.of(1, 1.0), Arguments.of(-1.0, -1.0), Arguments.of(-6.0, -1.0), Arguments.of(-23.8, -1.0), Arguments.of(25.5, 16581.375), Arguments.of(5945.8, 2.1019911829991202E11), Arguments.of(0.34, 0.039304));
```

You should remove that trailing 1. I don't know if java will always output it, so I think this is the better option.